fileio/ft_read_event.m

```diff
@@ -19,6 +19,8 @@
 %   'tolerance'      tolerance in samples when merging Neuromag analogue trigger channels (default = 1, meaning that an shift of one sample in both directions is compensated for)
 %   'blocking'       wait for the selected number of events (default = 'no')
 %   'timeout'        amount of time in seconds to wait when blocking (default = 5)
+%   'password'       password structure for encrypted data set (such as mayo_mef30 and mayo_mef21)
+%   'sortchannel'    = sort channel order according to either alphabet or number (default is alphabet if header is not provided, otherwise use header.SortChannel)
```

Comment:
channels are not part the output of ft_read_event, so I do not understand why you would want to sort on them

Reply:
We'll discuss whether to keep 'sortchannel' option in ft_read_event. Thanks.